Packs/VaronisSaaS/Integrations/VaronisSaaS/VaronisSaaS.py

```diff
@@ -1991,6 +1992,100 @@ def varonis_close_alert_command(client: Client, args: dict[str, Any]) -> bool:
                                 argToList(args.get('alert_id'), separator='|'), note)
 
 
+def update_remote_system_command(client: Client, args: Dict[str, Any]) -> str:
+    """update-remote-system command: pushes local changes to the remote system
+
+    :type client: ``Client``
+    :param client: XSOAR client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['data']`` the data to send to the remote system
+        ``args['entries']`` the entries to send to the remote system
+        ``args['incidentChanged']`` boolean telling us if the local incident indeed changed or not
+        ``args['remoteId']`` the remote incident id
+
+    :return:
+        ``str`` containing the remote incident id - really important if the incident is newly created remotely
+
+    :rtype: ``str``
+    """
+    parsed_args = UpdateRemoteSystemArgs(args)
+    alert_id = parsed_args.remote_incident_id
+
+    if not parsed_args.incident_changed or not alert_id:
+        return alert_id
+
+    if parsed_args.delta:
+        demisto.debug(f'Got the following delta keys {list(parsed_args.delta)}.')
+
+    demisto.debug(f'Sending incident with remote ID [{alert_id}] to remote system. Status {parsed_args.inc_status}.')
+    demisto.debug(f'Got the following data {parsed_args.data}.')
+
+    if (
+        ('Status' in parsed_args.delta or 'CloseReason' in parsed_args.delta)
+        and (parsed_args.data.get('Status', '').lower() == 'closed' or parsed_args.inc_status == IncidentStatus.DONE)
+    ):
+        demisto.debug(f'Closing remote incident {alert_id}')
+        note = parsed_args.data.get('CloseNotes', 'Closed from XSOAR')
+        close_reason = parsed_args.data.get('CloseReason', '').lower()
+        close_reason_id = CLOSE_REASONS.get(close_reason, CLOSE_REASONS['other'])
+        if not close_reason_id:
+            close_reason_id = CLOSE_REASONS['other']
+        varonis_update_alert(
+            client,
+            close_reason_id,
+            ALERT_STATUSES['closed'],
+            argToList(alert_id),
+            note
+        )
+
+    elif (
+        'Status' in parsed_args.delta
+        and parsed_args.data.get('Status').lower() != 'closed'
+        or parsed_args.inc_status == IncidentStatus.ACTIVE
+    ):
+        demisto.debug(f'Update remote incident {alert_id}')
+        note = 'Status changed from XSOAR'
+        status = parsed_args.data.get('Status', 'action required').lower()
+        status_id = ALERT_STATUSES.get(status)
+
+        close_reason_id = CLOSE_REASONS['none']
+        varonis_update_alert(
+            client,
+            close_reason_id,
+            status_id,
+            argToList(alert_id),
+            note
+        )
```

Comment:
In the case that a user updated the status of the incident in XSOAR to "closed", followed by closing the incident in Varonis, we would also like that the incident in XSOAR will be closed as well.
In order to do that you can try returning a close entry like [here](https://github.com/demisto/content/blob/229b7de1c9cadf11b030671c636a56a0a7a80d35/Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py#L2675) instead of the alert_id. The example is from a mirroring in function, but it might work here as well.
Another option is to create another post processing rule that will close the incident in XSOAR after the field status has been changed.

Reply:
Not sure what exactly need to be returned, I tried 
`
{
                'Type': EntryType.NOTE,
                'Contents': {
                    'dbotIncidentClose': True,
                    'closeNotes': note,
                },
                'ContentsFormat': EntryFormat.JSON
            }
`
and it's not working

Reply:
Also, post processing script in my case running only when I close an incident, it's not triggering in other cases. Maybe I need to add another tag to configuration?

Reply:
You can try to return a list of the entries like:
`[{ 'Type': EntryType.NOTE, 'Contents': { 'dbotIncidentClose': True, 'closeNotes': note, }, 'ContentsFormat': EntryFormat.JSON }]`
If it's not working let's schedule a meeting to discuss the issue.

Reply:
I tried, the last one, it's not working. You can schedule any time 11 a.m. - 6 p.m. from Monday till Friday.

Reply:
Updated REDME file of the Integration